graphql_compiler/compiler/sql_context_helpers.py

```diff
@@ -20,6 +20,14 @@ def get_node_selectable(node, context):
     return selectable
 
 
+def get_node_at_path(query_path, context):
+    """Return the SqlNode associated with the query path."""
+    if query_path not in context.query_path_to_node:
+        raise AssertionError(u'Unable to find SqlNode for query path {}'.format(query_path))
```

Comment:
Consider including the value of `context.query_path_to_node` and perhaps the entire `context`, for easier debugging.

Reply:
Good suggestion, I added context dumping to all these errors, it is helpful.